bluesky/plan_stubs.py

```diff
@@ -342,6 +342,15 @@ def rd(obj, *, default_value=0):
         The "single" value of the device
 
     """
+    # Location is canonical if it exists
```

Comment:
I think this should be added to the docstring above along with the other descriptions about how a read key is picked for the return value.

Something like, for example:
```
For devices that implement the Locatable protocol, the location is canonical and is returned without parsing the read keys.
```